client/gutenberg/extensions/wordads/edit.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * External dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import classNames from 'classnames';
+import { Component } from '@wordpress/element';
+import { PanelBody, Placeholder, SelectControl } from '@wordpress/components';
+import { InspectorControls } from '@wordpress/editor';
+
+const AD_FORMATS = [
+	{
+		name: __( 'Rectangle 300x250' ),
+		tag: '300x250_mediumrectangle',
```

Comment:
It's probably better that we use tag names like `mrec`, `leaderboard`, `mobile_leaderboard`, and `wideskyscraper` since those map directly into the keys of the `$ad_tag_ids` array in the WordAds module.